addon/macros/group-by.js

```diff
@@ -8,7 +9,7 @@ export default function groupBy(collection, property) {
   var dependentKey = collection + '.@each.' + property;
 
   return computed(dependentKey, function() {
-    var groups = [];
+    var groups = new A();
```

Comment:
Need to use Ember.A() because prototype extensions are now turned off by default. Or, need to not use `findBy` on the `groups` array.